src/cluster_slot_stats.c

```diff
@@ -148,10 +148,11 @@ static void clusterSlotStatsUpdateNetworkBytesOutForReplication(long long len) {
     client *c = server.current_client;
     if (c == NULL || !canAddNetworkBytesOut(c)) return;
 
+    uint64_t rlen = len*listLength(server.replicas);
     serverAssert(c->slot >= 0 && c->slot < CLUSTER_SLOTS);
     serverAssert(nodeIsPrimary(server.cluster->myself));
-    if (len < 0) serverAssert(server.cluster->slot_stats[c->slot].network_bytes_out >= (uint64_t)llabs(len));
-    server.cluster->slot_stats[c->slot].network_bytes_out += (len * listLength(server.replicas));
+    if (len < 0) serverAssert(server.cluster->slot_stats[c->slot].network_bytes_out >= -rlen);
```

Comment:
```suggestion
    serverAssert(server.cluster->slot_stats[c->slot].network_bytes_out - rlen >= 0);
```

Reply:
Same as other change suggestion: We need to only treat negative numbers in this assert case, not positive numbers.

Reply:
This makes the assert clear IMO:

```suggestion
    if (rlen < 0) {
        /* Decrement bytes added earlier. We should not reach below zero. */
        serverAssert(server.cluster->slot_stats[c->slot].network_bytes_out + rlen >= 0);
    }
```